src/Storage/Device/S3.php

```diff
@@ -255,6 +259,24 @@ public function delete(string $path, bool $recursive = false): bool
         return true;
     }
 
+    /**
+     * Delete files in given path, path must be a directory. Return true on success and false on failure.
+     *
+     * @see http://php.net/manual/en/function.filesize.php
+     *
+     * @param string $path
+     *
+     * @return bool
+     */
+    public function deletePath(string $path): bool
+    {
+        $uri = ($path !== '') ? '/' . \str_replace('%2F', '/', \rawurlencode($path)) : '/';
+
+        $this->call(self::METHOD_DELETE, $uri);
+
+        return true;
+    }
```

Comment:
Where would it return false on failure?

Looks like it will throw an exception 👍🏻

Reply:
Indeed the call method will throw an exception. Do you think we should handle the exception internally and return false on failure?

Reply:
I think it's fine as-is, but maybe we add a `@throws` block to the docblock to clarify?

Reply:
done!